x-pack/plugins/case/server/routes/api/cases/post_case.ts

```diff
@@ -73,6 +76,7 @@ export function initPostCaseApi({
           body: CaseResponseRt.encode(
             flattenCaseSavedObject({
               savedObject: newCase,
+              caseConfigureConnector,
```

Comment:
I am confused with this piece of code, @stephmilovic. Let me tell you what I think